servicex/cache.py

```diff
@@ -140,6 +138,11 @@ def __init__(self, cache_path: Path, ignore_cache: bool = False):
         self._path = cache_path
         self._ignore_cache = ignore_cache
 
+        # old windows versions have a limit of file path of 260 chars.
```

Comment:
What does this have to do with Kafka?

Reply:
it does not, but the test was failing so I fixed it.